software/hmmer/hmmalign/functions.nf

```diff
@@ -0,0 +1,60 @@
+/*
+ * -----------------------------------------------------
```

Comment:
Need to replace this file with the one [here](https://github.com/nf-core/modules/blob/master/software/lib/functions.nf ) since it has now been updated.

Reply:
`diff` says they're identical -- did I get the wrong version? (I got the master version.)

Reply:
Sorry, my mistakes, I didn't see the full path you wrote.